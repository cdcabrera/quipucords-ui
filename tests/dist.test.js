const { execSync } = require('child_process');

describe('Distribution', () => {
  it('should match a specific file output', () => {
    const output = execSync('find ./dist -type f -print0 | xargs -0');

    const replacedGeneratedFilesMinsHash = output
      .toString()
      .replace(/^\s+|\s+|\n+|\r+$/g, '')
      .replace(/\.\/dist/gi, '~./dist')
      .replace(/\.([a-z0-9]+)\./gi, '*')
      .split('~')
      .sort();

    expect(replacedGeneratedFilesMinsHash).toMatchSnapshot('dist-output');
  });
});
