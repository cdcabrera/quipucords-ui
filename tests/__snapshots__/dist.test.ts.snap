// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Build distribution should match a specific file output: dist-output 1`] = `
[
  "",
]
`;
